### ISSUE TITLE
"Failed to convert image" (with 320x240 PNG image)

### ISSUE THREAD

**Issue by dasebasto** [marian-m12l/studio#42]
This project is amazing, kudos to Marian!

I get the following exception when trying to transfer a custom pack to the device:

```
mars 31, 2020 3:39:02 PM studio.webui.service.LibraryService
SEVERE: Failed to convert archive format pack to binary format
java.io.IOException: Failed to convert image
        at studio.core.v1.utils.ImageConversion.convertImage(ImageConversion.java:37)
        at studio.core.v1.utils.ImageConversion.pngToBitmap(ImageConversion.java:21)
        at studio.core.v1.utils.PackAssetsCompression.withUncompressedAssets(PackAssetsCompression.java:92)
        at studio.webui.service.LibraryService.getBinaryPackFile(LibraryService.java:134)
        at studio.webui.api.DeviceController$1.run(DeviceController.java:66)
        at java.base/java.util.TimerThread.mainLoop(Timer.java:556)
        at java.base/java.util.TimerThread.run(Timer.java:506)
mars 31, 2020 3:39:02 PM studio.webui.api.DeviceController
SEVERE: Failed to read or convert pack
mars 31, 2020 3:39:02 PM studio.webui.MainVerticle
SEVERE: Exception thrown
io.vertx.core.impl.NoStackTraceThrowable: Failed to read or convert pack
```

The image causing the bug is a PNG file, created by screenshot utility ShareX: https://i.imgur.com/SVPVASy.png

Replacing the image in the pack by the same image in BMP format solves the issue.

**Comment by marian-m12l** [comment]
Hi, thanks for the feedback!
The issue is caused by the PNG image containing an alpha layer. This should be fixed on `master` now (by redrawing images without alpha layer; transparency is replaced by a black background).

**Comment by dasebasto** [comment]
Great job!
I imagined it could have something to do with transparency :)

**Comment by marian-m12l** [comment]
fixed with release 0.1.13
